src/index.js

```diff
@@ -82,12 +85,26 @@ export default function supernova(env) {
         if (!layout) {
           return Promise.resolve();
         }
+
+        const themeService = createThemeService({
+          theme: Theme,
+          config: {
+            id: "orgChart",
+            resolve: [
+              ["object.orgChart", "axis.label.name", "fontSize", "14px"],
+              ["object.orgChart", "label.value", "fontSize", "11px"],
+            ],
+          },
+        });
+
+        const styleModel = createStyleModel({ layout, themeService, flags });
```

Comment:
I'm thinking about if this is a good name or not. As soon as you hear model, I at least think of the  `useModel` model. I don't know, maybe `resolvedStyleProperties`? But that isn't really true since there are a couple more steps "resolving" the styling. @niekvanstaveren any thoughts on this, and the I guess the structure as well, having this first step, and then calling `cardStyling`?

Reply:
I don't actually have an issue with the name. However I think the structure becomes complex/confusing. I don't think we need two different objects taking care of the styling.
The styleModel is used as some intermediate to create the cardStyling. Probably not needed to have those multiple steps indeed.

Reply:
So the idea would be to move all the logic in styleModel to cardStyling?